packages/client/src/api.ts

```diff
@@ -12,40 +12,82 @@ import { MessagingChannelBase } from './base'
 import { handleNotFound } from './errors'
 
 export abstract class MessagingChannelApi extends MessagingChannelBase {
+  /**
+   * Creates a new client
+   * @param id Id of the client
+   * @returns Id and token of the created client
+   */
   async createClient(id?: uuid): Promise<{ id: uuid; token: string }> {
     return (await this.http.post('/admin/clients', { id }, { headers: this.adminHeader })).data
   }
 
+  /**
+   * For internal usage. Assures that a client with the provided name and token exists
+   */
   async syncClient(config: { name: string; id?: uuid; token?: string }): Promise<{ id: uuid; token: string }> {
     return (await this.http.post('/admin/clients/sync', config, { headers: this.adminHeader })).data
   }
 
+  /**
+   * Configures a client with channel and webhook configs
+   * @param clientId id of the client to configure
+   * @param config channel and webhook configs
+   * @returns a list of webhook tokens
+   */
   async sync(clientId: uuid, config: SyncRequest): Promise<SyncResult> {
     return (await this.http.post('/sync', config, { headers: this.headers[clientId] })).data
   }
 
+  /**
+   * Polls health events for a client
+   * @param clientId id of the client to poll
+   * @returns a list of health events per channel
+   */
   async getHealth(clientId: uuid): Promise<HealthReport> {
     return this.deserializeHealth(
       (await this.http.get<HealthReport>('/health', { headers: this.headers[clientId] })).data
     )
   }
 
+  /**
+   * Creates a new messaging user
```

Comment:
PP: Messaging with a capital `m` as we refer to the service/product.

Reply:
nah